auth/auth0/test/index.js

```diff
@@ -0,0 +1,33 @@
+function displayAuthResults(accessToken, idToken) {
+  var el = document.getElementById('mutation');
+  el.style.visibility = 'visible';
+  var mutation = `
+  # Run this mutation in the Graphcool Playground to authenticate a user
+
+  mutation {
+    authenticateUser(
+      accessToken: "${accessToken}"
```

Comment:
For instant (with graphical auth0 integration), I only need to give the `id_token`. And the `accessToken` is in `idToken` in my memories 😉

Reply:
@fabien0102 That would be great yes, can you let me know which grants your integration uses ?

Reply:
![image](https://user-images.githubusercontent.com/1761469/31943749-014969bc-b8ca-11e7-8b83-1fd9c1c74d12.png)
The legacy one :wink:

Reply:
That's not the case. The built-in integration used HS256 tokens and the non-OIDC flow. So that doesn't apply here. 
You can read in the Auth0 documentation that the id_token should not be used for authentication.

Reply:
In the end I'm not going to need the accessToken since I'm removing the profile fetch from Auth0